### ISSUE TITLE
Appium Send Keys is not sending the right keystrokes (depending on keyboard layout and uppercase letters being replaced)

### ISSUE THREAD

**Issue by CRYPTO-7CA** [appium/appium-desktop#2134]
## Appium or Appium Desktop?

This Problem happens in 2 Appium Clients in the exact same manners. (1) Appium Library for Python:

- robotframework-appiumlibrary   2.0.0
- Appium-Python-Client           2.7.1
- robotframework-seleniumlibrary 6.0.0
- robotframework                 6.0.1

And with Appium Inspector 2022.11.1

To automate GUI interaction on Windows I use WinAppDriver. Tried it on version 1.2 RC and 1.2.1 - as for version 1.2.99 WinAppDriver would not load up. The Application under test was developed using Qt with pure C++ QWidget (without QML).

Finally I installed the Appium server through Node.js ("npm i -g appium"). It's version used is 1.22.3.

I see, that Appium Server has received the string correctly.
![image](https://user-images.githubusercontent.com/61016421/205693881-c79ee91d-92d2-4d1c-8393-764cb4dd3186.png)

- [ ? ] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

I have the exact same problems as was reported here on Stackoverflow over 2 years ago -> https://stackoverflow.com/questions/60948389/appium-sendkeys-to-wpf-uses-different-keyboard-layout

In short if the currently selected keyboard layout on my machine is DEU (German) the keystrokes transform as follows:

"y" becomes "z" and conversely "z" becomes "y".
Also "-" becomes "ß" but "ß" stays "ß"

On top of that if I send a string with multiple uppercase letters e.g. "boyFriENd" all capital letters are replaced with the first capital letter, so the result would be "bozFriFFd".

![image](https://user-images.githubusercontent.com/61016421/205692466-2cca6192-22c1-432e-8bf0-3849513b3558.png)

## Environment

- I am running Appium Inspector 2022.11.1
- Appium (server) 1.22.3
- WinAppDriver 1.2, 1.2.1
- (also check Python Library mentioned above)
- I am on Windows 10 Enterprise

**Comment by jlipps** [comment]
you reported this issue to the repo for appium desktop, which is not supported.

you should upgrade to appium 2 and the latest versions of all drivers, and if you still notice an issue, create a bug report at the main appium repo.

but at the end of the day this looks like a WinAppDriver issue so the appium team is not going to be able to assist. unfortunately microsoft has not chosen to maintain WinAppDriver.

**Comment by CRYPTO-7CA** [comment]
In case anyone wonders, I got Appium2 and Windows driver now installed (Appium2 through npm and drivers through appium CLI) and have retried sending keystrokes. However Windows Application Driver 1.2.1 was reused (could not find any substitute for it).
This issue persists but I suspect it's most likely a Problem with WinAppDriver and not Appium (Appium2) or it's Windows drivers.
